proof/crefine/RISCV64/SR_lemmas_C.thy

```diff
@@ -13,6 +13,22 @@ begin
 
 context begin interpretation Arch . (*FIXME: arch_split*)
 
+section "vm rights"
+
+lemma vmRightsToBits_bounded:
+  "vmRightsToBits rights < 4"
+  by (cases rights; clarsimp simp: vmRightsToBits_def)
+
+lemma vmRightsToBits_not_0:
+  "vmRightsToBits rights \<noteq> 0"
```

Comment:
it's sometimes easier to work with `0 < n` than `n \<noteq> 0`

Reply:
This one is just moved from further down resolving a FIXME. I don't think we should touch the statement at this point.